catalogue/webapp/pages/workv2.js

```diff
@@ -303,12 +554,13 @@ export const WorkPage = ({
   );
 };
 
-WorkPage.getInitialProps = async (context) => {
+WorkPage.getInitialProps = async (context,  { toggles = {} }: ExtraProps) => {
   const {id} = context.query;
   const {asPath} = context;
   const queryStart = asPath.indexOf('?');
   const previousQueryString = queryStart > -1 && asPath.slice(queryStart);
   const work = await getWork({ id });
+  const showRedesign = Boolean(toggles.showWorkRedesign);
```

Comment:
You might want to pull master as the toggle [are now attached to the `ctx.query.toggles`](https://github.com/wellcometrust/wellcomecollection.org/blob/master/catalogue/webapp/pages/_app.js#L30-L31)